app/models/guide_edition.rb

```diff
@@ -14,6 +14,6 @@ def has_video?
   end
 
   def safe_to_preview?
-    parts.any? and parts.first.slug.present?
+    parts.any? && parts.first.slug.present? && !edition.archived?
```

Comment:
I changed this to be `super && [guide stuff]`, so it gets `!archived?` from Edition.